### ISSUE TITLE
"Donate today!": broken link https://donate.pypi.io/

### ISSUE THREAD

**Issue by haypo** [pypa/warehouse#2423]
The donate.pypi.io domain doesn't exist:

```
$ host donate.pypi.io
Host donate.pypi.io not found: 3(NXDOMAIN)
```

But https://pypi.org/ homepage has a "Donate today!" link to https://donate.pypi.io/

**Comment by Siecje** [comment]
It redirects to `https://psfmember.org/civicrm/contribute/transact?reset=1&id=13`

**Comment by di** [comment]
Seems OK to me:

```
$ host donate.pypi.io
donate.pypi.io is an alias for http2.python.map.fastly.net.
http2.python.map.fastly.net has address 151.101.1.63
http2.python.map.fastly.net has address 151.101.65.63
http2.python.map.fastly.net has address 151.101.129.63
http2.python.map.fastly.net has address 151.101.193.63
http2.python.map.fastly.net has IPv6 address 2a04:4e42::319
http2.python.map.fastly.net has IPv6 address 2a04:4e42:200::319
http2.python.map.fastly.net has IPv6 address 2a04:4e42:400::319
http2.python.map.fastly.net has IPv6 address 2a04:4e42:600::319
```

But we should probably change the link to be `donate.pypi.org` now that we have that domain, so I made #2424.

**Comment by haypo** [comment]
Strange. Now it works. Maybe someone fixed the domain since I reported my issue?

**Comment by ewdurbin** [comment]
resolved in #2424
